Neos.Neos/Classes/View/FusionView.php

```diff
@@ -78,14 +78,18 @@ class FusionView extends AbstractView
     public function render()
     {
         $currentNode = $this->getCurrentNode();
-        $currentSiteNode = $currentNode->getContext()->getCurrentSiteNode();
+        $currentSiteNode = $this->getCurrentSiteNode();
         $fusionRuntime = $this->getFusionRuntime($currentSiteNode);
 
-        $dimensions = $currentNode->getContext()->getDimensions();
-        if (array_key_exists('language', $dimensions) && $dimensions['language'] !== []) {
-            $currentLocale = new Locale($dimensions['language'][0]);
-            $this->i18nService->getConfiguration()->setCurrentLocale($currentLocale);
-            $this->i18nService->getConfiguration()->setFallbackRule(['strict' => false, 'order' => array_reverse($dimensions['language'])]);
+        if ($currentNode instanceof LegacyNodeInterface) {
+            $dimensions = $currentNode->getContext()->getDimensions();
+            if (array_key_exists('language', $dimensions) && $dimensions['language'] !== []) {
+                $currentLocale = new Locale($dimensions['language'][0]);
+                $this->i18nService->getConfiguration()->setCurrentLocale($currentLocale);
+                $this->i18nService->getConfiguration()->setFallbackRule(['strict' => false, 'order' => array_reverse($dimensions['language'])]);
+            }
+        } else {
+            // TODO: special case for Language DimensionSpacePoint!
```

Comment:
can we ignore this case for now?

Reply:
@bwaidelich yes (for now), but very very good catch anyways: This method is overridden completely with "our" EventSourcedNeosAdjustments FusionView::render() - so I remove these adjustments here.

Reply:
done!